cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -35,6 +34,13 @@
 ]
 
 
+def remove_vm(provider, vm_name):
+    if provider.mgmt.does_vm_exist(vm_name):
+        if provider.mgmt.is_vm_running(vm_name):
+            provider.mgmt.stop_vm(vm_name)
+        provider.mgmt.delete_vm(vm_name)
```

Comment:
No, it wasn't, because in 5.7.1 VMs on RHV provider are started after provisioning by default, which means it can't be deleted before it's stopped.
There are 2 reasons why I didn't put this into cleanup_vm:
1. cleanup_vm catches the exception and hides the information in logger.warning, where we can't see it, because the test where it failed to delete a VM didn't fail. This causes other tests to fail and it's hard to investigate, because the root problem is not seen in any of the tests which failed.
2. I don't know whether I can use does_vm_exist, is_vm_running, stop_vm in cleanup_vm, because it's used by other providers, and I'm not sure whether they have these methods defined